### PR TITLE
Im 160

### PR DESCRIPTION
**Pull request by mrstarware**

### CODE REVIEW

File: config/StoredProcedures/EditUserByUsername [hunk 1]
```diff
@@ -0,0 +1,15 @@
+use imp_db_dev;
```
Comment (johnschroeder) [inline config/StoredProcedures/EditUserByUsername]: Please remove this line

Comment (mrstarware) [inline config/StoredProcedures/EditUserByUsername]: Ok I pulled the most recent version of master w/ your SQL changes. Was there anything else?

Comment (NamespaceValentine) [inline config/StoredProcedures/EditUserByUsername]: This line still needs to be removed.

Comment (mrstarware) [inline config/StoredProcedures/EditUserByUsername]: I thought that did change. Weird. Ok. It should be deleted now.

**Pull request merged**